.appveyor.yml

```diff
@@ -57,6 +57,17 @@ install:
       $env:path += ";C:\ponyc-windows-libs\lib"
       cd C:\projects\ponyc
       C:\premake5.exe --with-tests --to=work/vs2015 vs2015
+  - ps: |
+      $file_version = (Get-Content "C:\projects\ponyc\VERSION")
+      Update-AppveyorBuild -Version "$file_version.${env:appveyor_build_number}"
+
+after_build:
+  - ps: |
+      cd C:\projects\ponyc
+      7z a -tzip "ponyc-win64-${env:appveyor_build_version}-${env:configuration}-llvm-${env:llvm}.zip" "packages\" "build\$env:configuration\ponyc.*" "build\$env:configuration\ponyrt.*"

```

Comment:
Good question.  I think that to support 32-bit builds we need to cover things like [this for LLVM](https://github.com/ponylang/ponyc-windows-llvm/blob/master/compilellvm.ps1#L16), [this for PCRE](https://github.com/killerswan/ponyc/blob/06bd14ead9c4d03f85fd2c7cc768313140ee5c73/.appveyor.yml#L47), and [this in the premake script](https://github.com/killerswan/ponyc/blob/06bd14ead9c4d03f85fd2c7cc768313140ee5c73/premake5.lua#L156), so I haven't experimented with 32-bit anything quite yet.